samples/name_domains_values/Coded_Domain_Reinforced.csv

```diff
@@ -0,0 +1,4 @@
+Name,Value
++,0
+-,1
++/-,2
```

Comment:
I went ahead and included the Reinforced domain as @csmoore suggested it, however the values of 0, 1, and 2 for these are not represented in the standard, and certainly not included in the SIDC, therefore probably not included within any typical military messages or transmitted as 0, 1, and 2 by any systems.that I am aware of.  

I think a better alternative would be:

|Name | Value|
|--------|-----------|
|Reinforced | + |
|Reduced | - |
|Reinforced and Reduced | +/- |